rclpy/rclpy/qos.py

```diff
@@ -270,6 +270,9 @@ class QoSHistoryPolicy(QoSPolicyEnum):
     RMW_QOS_POLICY_HISTORY_KEEP_ALL = 2
     KEEP_ALL = RMW_QOS_POLICY_HISTORY_KEEP_ALL
 
+#Alias with the old name, for retrocompatibility
+HistoryPolicy = QoSHistoryPolicy
```

Comment:
```python3
QoSHistoryPolicy = HistoryPolicy
```
And change the class name from `QosHistoryPolicy` to `HistoryPolicy`.

I know that it looks like the same, but in this way the old name can be easily deprecated/deleted later.

Reply:
extremly sorry for this silly mistake..

Reply:
I have made the changes required..please have a look to those

Reply:
You don't have to sorry, it's ok to make mistakes.